scripts/kick-off.js

```diff
@@ -36,7 +36,7 @@ const config = {
 
 const deleteFolderRecursive = path => {
   if (fs.existsSync(path)) {
-    fs.readdirSync(path).forEach((file, index) => {
+    fs.readdirSync(path).forEach((file) => {
```

Comment:
:thinking: why doesn't the linter work for this case?  `forEach((file) =>`

Reply:
we deleted that rule

Reply:
why? :crying_cat_face:

Reply:
done